src/main/kotlin/org/malachite/estella/queues/ExampleConsumer.kt

```diff
@@ -0,0 +1,33 @@
+package org.malachite.estella.queues
+
+import com.rabbitmq.client.AMQP
+import com.rabbitmq.client.DefaultConsumer
+import com.rabbitmq.client.Envelope
+
+class ExampleConsumer() : Queue() {
+
+    var received = 0
+
+    init {
+        channel.queueDeclare("hello", false, false, false, null)
+        println(" [*] Waiting for messages. To exit press CTRL+C")
+        val consumer = object : DefaultConsumer(channel) {
+            override fun handleDelivery(
+                consumerTag: String,
+                envelope: Envelope,
+                properties: AMQP.BasicProperties,
+                body: ByteArray
+            ) {
+                val message = String(body, charset("UTF-8"))
+                received+=1
+                println(" [x] Received '$message'")
+            }
+        }
+        channel.basicConsume("hello", true, consumer)
```

Comment:
Basic consume pobiera wartość cały czas, czy tylko raz? Ponadto spring amqp daje Ci piękną adnotację (jak w sumie cały spring na adnotacjach stoi), więc niech przykładowy faktycznie będzie przykładem: https://www.baeldung.com/spring-amqp (z tego tutoriala o punkt 5.5 mi chodzi - będzie zwięźlej).

Reply:
Znaczy ogólnie dałem tylko jedną wiadomość ponieważ mimo wszystko mamy limit wiadomości przesyłanych po rabicie milion to sporo wiadomo, ale no. I tak basic consume pobiera cały czas.